.github/workflows/kind.yaml

```diff
@@ -0,0 +1,112 @@
+name: Kind
```

Comment:
Maybe a more concrete name?

Reply:
Suggestion? I like short names in general and this is testing Kind integration :-)

Reply:
What do you think about `KindConnectivity` / `connectivity.kind.yaml`?
I suspect we will have many `connectivity.foo.yaml`, so I'm thinking we might as well start to categorize stuff with filenames, since IIRC subfolders are not supported by GH Actions.